src/test/kotlin/it/unibo/protelis/web/execution/simulated/NewSimulatedProtelisEngineTest.kt

```diff
@@ -0,0 +1,63 @@
+package it.unibo.protelis.web.execution.simulated
+
+import it.unibo.protelis.web.execution.ProtelisObserver
+import it.unibo.protelis.web.execution.ProtelisUpdateMessage
+import java.util.concurrent.CompletableFuture
+import kotlinx.io.errors.IOException
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+@DisplayName("Test simulated Protelis engine with Alchemist")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [UndocumentedPublicClass - [NewSimulatedProtelisEngineTest] at /src/src/test/kotlin/it/unibo/protelis/web/execution/simulated/NewSimulatedProtelisEngineTest.kt:13:1](https://app.codacy.com/manual/NiccoMlt/Protelis-Web/pullRequest?prid=5040950)